src/main/java/com/cascadebot/cascadebot/commandmeta/ICommandExecutable.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package com.cascadebot.cascadebot.commandmeta;
+
+import net.dv8tion.jda.core.entities.Member;
+
+public interface ICommandExecutable {
+
+    public void onCommand(Member sender, CommandContext context);
+
+    public String command();
```

Comment:
Souldn't this be default command as the commands can change?

Reply:
Reason I named this like it is is because subcommands use the same function, it seems odd to me to have subCommand.defaultCommand() when their command can't change

Reply:
Ok. I'm fine with it being named this. feel free to merge